pkg/daemon/ceph/osd/volume.go

```diff
@@ -696,3 +696,30 @@ func getCephVolumeRawOSDs(context *clusterd.Context, clusterName string, cephfsi
 
 	return osds, nil
 }
+
+func callCephVolume(context *clusterd.Context, args ...string) (string, error) {
+	// Use stdbuf to capture the python output buffer such that we can write to the pod log as the
+	// logging happens instead of using the default buffering that will log everything after
+	// ceph-volume exits
+	baseCommand := "stdbuf"
+	// TODO: what if log path were /dev/stderr and we didn't do combined output so that logs would
+	//       appear in the prepare pod log?
```

Comment:
Open question. Thoughts @travisn / @leseb ?

Reply:
Sounds good for the logs to be piped to the same pod log